### PR TITLE
Adding examples to spam doc of akismet/recaptcha configs to add to staticman.yml

### PR DESCRIPTION
**Pull request by RyanTG**
Happy to revise this, based on your review.

### CODE REVIEW

File: docs/spam.md [hunk 1]
```diff
@@ -23,15 +23,37 @@ Another effective method of combating spam is to use reCAPTCHA. If you enable Ak
...
+  ``` yml
+    reCaptcha:
+      enabled: true
+      # siteKey and secret should match your config.toml
+      siteKey: ""
+      # (!) ENCRYPT reCaptcha secret key using Staticman /encrypt endpoint
+      # i.e. https://{your-staticman-api}/v3/encrypt/{your-site-secret}
+      # For more information, visit https://staticman.net/docs/encryption
+      secret: ""
+  ```
```
Comment (alexwaibel) [inline docs/spam.md]: This is the same as the current step 2:
> 2. Use the Staticman encryption endpoint to encrypt the secret and add the reCAPTCHA options to your site config

Can move this example yaml up so it's right under that step?

Comment (alexwaibel) [reply docs/spam.md]: Also, which config is `config.toml` referring to?

File: docs/spam.md [hunk 2]
```diff
@@ -23,15 +23,37 @@ Another effective method of combating spam is to use reCAPTCHA. If you enable Ak
...
+1. Add the Akismet config to your staticman.yml file, making sure that the values below match your form's fields
+  ``` yml
+  # Akismet spam detection
+    akismet:
+      enabled: true
+      author: "name"
+      authorEmail: "email"
+      authorUrl: "website"
+      content: "message"
+  ```
```
Comment (alexwaibel) [inline docs/spam.md]:
```suggestion
1. Add the Akismet config to your staticman.yml site config file, making sure that the values below match the name properties of your form's fields
  ``` yml
    akismet:
      enabled: true
      author: "name"
      authorEmail: "email"
      authorUrl: "website"
      content: "message"
  ```
```
nit

**Review by alexwaibel — CHANGES_REQUESTED** [review]
Thank you for submitting this. I agree that inline examples here would be helpful. I've left some small suggestions.

File: docs/spam.md (hunk 2 above)
Comment (RyanTG) [reply docs/spam.md]: Oops, one minute. I missed this review note.

File: docs/spam.md (hunk 1 above)
Comment (RyanTG) [reply docs/spam.md]: I believe `config.toml` a hugo-specific reference. I removed it. And the other comment about encrypting the secret was redundant with the text in step 2, so I removed that too.

I haven't implemented reCAPTCHA on my own jekyll/staticman sites, so I'm not sure if these instructions could be further clarified.

**Comment by RyanTG** [comment]
I believe I addressed everything. I also fixed a link to the encryption instructions.

I struggled a bit to get the code snippets to be correctly-formatted. 1) They look kinda ugly on the page, because there's an extra border being added (at least in Firefox), and 2) The indentation in the first line is off. It's like it really wants to force an indentation on the first line. In the code you can clearly see that I do not want to indent the first line. But I think it's good enough for now.

**Comment by alexwaibel** [comment]
Thanks again for submitting this and for the speedy revision. The Firefox double borders on code blocks are something I've seen as well but just haven't had time to dig into. I agree they're ugly but at least all they seem to function fine otherwise. I'll open an issue for this to keep track of it.

**Pull request merged**